sql_generators/baseline_clients_city_seen_v1/templates/query.sql

```diff
@@ -0,0 +1,230 @@
+ -- Query generated via sql_generators.clients_city_seen.
+ -- this mimics the logic used in baseline_clients_daily_v1.
+{% raw %}
+{% if is_init() %}
+{% endraw %}
+WITH
+{% for app_id in app_id_list -%}
+  base_{{ app_id }} AS (
+  SELECT
+    submission_timestamp,
+    DATE(submission_timestamp) AS submission_date,
+    LOWER(client_info.client_id) AS client_id,
+    sample_id,
+    mozfun.glean.parse_datetime(ping_info.end_time) AS parsed_end_time,
+    `moz-fx-data-shared-prod.udf.glean_timespan_seconds`( metrics.timespan.glean_baseline_duration ) AS duration,
+    metadata.geo.city,
+    metadata.geo.subdivision1 AS geo_subdivision1,
+    metadata.geo.subdivision2 AS geo_subdivision2,
+  FROM
+    `{{ project_id }}.{{ app_id }}_stable.baseline_v1`
+  WHERE
+    client_info.client_id IS NOT NULL
+    AND sample_id = 0
+    AND DATE(submission_timestamp) <= "2025-08-25"
```

Comment:
Current date is probably more appropriate for the init so it gets everything

```suggestion
    AND DATE(submission_timestamp) <= CURRENT_DATE()
```